docs/concept-workflows.md

```diff
@@ -1,30 +1,209 @@
+The Workflow execution state is even preserved through crashes of the Temporal Server itself.
+
+## Freedom to relocate
+
+While there are a few specific use-cases that may require a Workflow to start and finish on the same host, Workflow code is completely independent from the location in which it is executed
+
+Instead of deploying executable code to a host directly, Temporal applications use special [Worker](/docs/concept-workers) processes whose sole job is to host and execute Workflow and Activity code.
+The Worker communicates with the Temporal Server in order to know which line of code to execute through [Tasks](/docs/concept-task-queues) it picks up.
+
+Because a Workflow is executed piece by piece, and because the Server tracks the state of that execution, there can be multiple Worker processes in multiple locations capable of executing any one of those pieces.
+
+Essentially load balancing and processing failover are "out-of-the-box" features of Temporal Workflows.
+
+## Workflows have options
+
+When you start a Workflow, you can pass along parameters that tell the Temporal Server how to handle the Workflow.
+This includes the ability to set timeouts for Workflow execution, a retry policy, the Task Queue name, a data converter, and child Workflow options.
+
+### Timeout settings
+
+It's often necessary to limit the amount of time a specific Workflow can be running.
+
+To support this, the following three parameters can be provided to Workflow options:
+
+#### Execution timeout
+
+- **Description**: This is the maximum amount of time that a Workflow should be allowed to run including retries and any usage of the "Continue-as-new" feature.
+This is different from [Run timeout](#run-timeout).
+- **Use-case**:
+
+#### Run timeout
+
+- **Description**: This is the maximum amount of time that a single Workflow run is restricted to.
+- **Use-case**:
+
+#### Task timeout
+
+- **Description**: This is the maximum amount of time that the Server will wait for the Worker to start processing a Task after the Task has been pulled from the Task Queue.
+If a [Command Task](/docs/glossary/#command) is lost, it is retried after this timeout.
+- **Use-case**:
+
+### Retry policy
+
+There may be scenarios where you need to retry a Workflow's execution from the very beginning.
+In this case, you can use supply a retry policy when you start the Workflow.
+
+#### Initial interval
+
+- **Description**: Amount of time that must elapse before the first retry occurs.
+- **Use-case**:
+
+#### Backoff coefficient
+
+- **Description**: Retries occur exponentially.
+The coefficient specifies how fast the retry interval is growing. The coefficient of 1 means that the retry interval is always equal to the `InitialInterval`.
+- **Use-case**:
```

Comment:
@mfateev ?

Reply:
When the coefficient is larger than 1 the retry interval grows with every retry. It allows quick retry initially which helps with short intermittent failures and a much larger retry interval for long-lasting outages. We recommend setting maximum retry interval to avoid retry intervals growing too much.